backend/src/EventSubscriber/JWTCreatedSubscriber.php

```diff
@@ -60,4 +77,17 @@ private function buildRoomPermission(User $user): string
 
         return '';
     }
+
+    private function getRoomFromRequest(User $user): ?string
```

Comment:
I don't think this name is explicit enough, `getRoomFromRequest` makes me think it always returns the room if it is found on the request, but it does not if certain conditions are not met. IMO we should either find a better naming or add a comment above this method explaining what this is doing (I would consider the first option)

Reply:
Comment added! Not very good finding names to functions ^^U

Reply:
@jordisala1991 changed the comment :)